docs/conf/prague/2021/sponsors/prospectus.rst

```diff
@@ -0,0 +1,282 @@
+:template: {{year}}/generic.html
+:banner: _static/conf/images/headers/{{ shortcode }}-group.png
+
+Online Sponsorship Prospectus
+#############################
+
+.. contents:: Sections
+   :local:
+   :depth: 1
+   :backlinks: none
+
+Introduction
+============
+
+Welcome to our brand new **Online Conference Sponsorship Prospectus**.
+This document has been adapted for our new virtual conference format,
+including new benefits and increased value across our community.
+
+We're excited to work with the organizations in our community to build the best possible event in {{ year }}.
+In particular, we would love your feedback on sponsorship levels and benefits.
+We expanded the offerings and introduced a lot of new things,
+but please let us know if there are other points of interaction with our community that would be valuable for you.
+
+Concept
+=======
+
+{% include "conf/sponsorship/concept.rst" %}
+
+Demographics
+============
+
+{% include "conf/sponsorship/demographics.rst" %}
+
+Why Sponsor
+===========
+
+{% include "conf/sponsorship/why.rst" %}
+
+Sponsorship Packages
+====================
+
+The following options are suggested sponsorship levels. We are happy to discuss adjustments and custom packages.
+
+Second Draft
+------------
+
+The **Second Draft** package is great for companies looking to hire or to promote a product.
+
+- Three (3) tickets_
+- Two (2) featured job postings on our Job Board, also promoted in our newsletter ({{ newsletter_subs }} subscribers)
+- A short description (250 characters) and logo of your company on the conference website
+- A table at our **virtual job fair**
+- Name included in welcome announcement in email newsletters and social media
+
+The **Second Draft** package costs **{{sponsorship.second_draft.price}}**.
```

Comment:
@ericholscher are we tweaking sponsorship prices this year?

Reply:
Set to 1,500, 3,000, 6,500, 12,500 as discussed